techniques/system/common/1.0/promises.st

```diff
@@ -190,7 +190,7 @@ bundle agent check_disable_agent
 
       # Only define this class when we're ready to die - this is a special class name in "abortclasses"
       "should_not_continue"
-        expression => "should_disable_agent",
+        expression => "should_disable_agent.!ignore_disable_agent",
```

Comment:
OK, after some thought, this *does* make sense, once you consider that "force_inventory" is a bit of a misnomer, but we're not going to change that now.

So, let's stick with ignore_disable_inventory and ignore_disable_agent class names.